content/api/_index.md

```diff
@@ -6,42 +6,42 @@ weight: 1
 Programming your robot is done in [Python](https://www.python.org/), specifically version {{% python_version %}}. You can learn more about Python from their [docs](https://docs.python.org/{{% python_version %}}/), and our [whirlwind tour](/tutorials/python-whirlwind-tour).
 
 ## Setup
-The following two lines are required to setup the robot.
+The following two lines are required to initialise the robot.
 
 ```python
 from robot import Robot
 
 r = Robot()
 ```
 
-This will setup your robot and allow it to send and receive data from the robot's various boards. Once this has been setup, this object can be used for most of the functions of the robot.
+This will initialise your robot and allow it to communicate with the robot's various boards. Once this has been done, this `Robot` object can be used for most of the functions of the robot.
 
 The remainder of the tutorials pages will assume your robot is defined as `r`.
```

Comment:
This sentence uses a rather different meaning of "your robot" than we do elsewhere. It would be good to fix that while we're here.

Reply:
Good spot - this sentence seems to be more or less a duplicate of the sentence preceding the code sample anyway.

Reply:
Oops, I mistakenly thought you were referring to the line changed in the diff rather than the line this thread is attached to. I've cleaned both up anyway.